modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue

```diff
@@ -141,24 +164,46 @@ export default {
       } else {
         return 'checkbox-blank-icon';
       }
-    },
-    more () {
-      const config = {
-        button: {
-          label: 'apostrophe:moreOperations',
-          iconOnly: true,
-          icon: 'dots-vertical-icon',
-          type: 'outline'
-        },
-        menu: Array.isArray(this.options.moreActions) ? this.options.moreActions : []
-      };
-
-      return config;
     }
   },
+  mounted () {
+    this.computeShowedOperations();
+  },
   methods: {
+    computeShowedOperations () {
```

Comment:
I just avoid using computed properties when I don't need it. In this case we only need to compute these values when component mounts and when a filter is applied. It would add a certain amount of data to watch for Vue which we can do without in this case. What do you mean by the `the property function`?

Reply:
That sounds fine. I've not read about the impact of using more computed properties for this kind of thing, but I see what you're saying.

Regarding moving `showOperation`, I meant that you might need to have that be directly part of `showedOperations`, not only used in it, to make the whole thing computed since `showOperation` contains the filter property references.